core/app/models/spree/promotion.rb

```diff
@@ -40,6 +40,7 @@ class Promotion < Spree::Base
         where(table[:expires_at].eq(nil).or(table[:expires_at].gt(time)))
     end
     scope :applied, -> { joins(:order_promotions).uniq }
+    scope :applied_automatically, -> { where(apply_automatically: true) }
```

Comment:
I'm actually considering removing the scope and just calling `where(apply_automatically: true)` from `Spree::PromotionHandler::Cart#sale_promtions` as it's the only place that actually cares about it.